site/docs/build-ref.html

```diff
@@ -357,6 +357,29 @@ <h4>Package names, <code>//<b>package-name</b>:...</code></h4>
   end with a slash.
 </p>
 
+<h4 id="pattern_expansion">Target pattern expansions, <code>//<b>...</b></code>
```

Comment:
https://docs.bazel.build/versions/4.0.0/guide.html#specifying-targets-to-build also mentions this, and also properly defines `target_pattern`. Link there when introducing the term?

Reply:
After looking at the guide, I decided to remove this little section. I think that's a much better reference.

Instead, I added two entries in the guide to be more explicit about these two patterns. Let me know what you think.

Thanks!